### PR TITLE
Auto restart on language change

### PR DESCRIPTION
**Pull request by nc-kano**

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19/ViewModels/SettingsGeneralViewModel.cs [hunk 1]
```diff
@@ -10,21 +10,21 @@ namespace NDB.Covid19.ViewModels
...
-        public static string SETTINGS_GENERAL_NN = "SETTINGS_GENERAL_NN".Translate();
-        public static string SETTINGS_GENERAL_NB = "SETTINGS_GENERAL_NB".Translate();
+        public static string SETTINGS_GENERAL_MOBILE_DATA_HEADER => "SETTINGS_GENERAL_MOBILE_DATA_HEADER".Translate();
+        public static string SETTINGS_GENERAL_MOBILE_DATA_DESC => "SETTINGS_GENERAL_MOBILE_DATA_DESC".Translate();
+        public static string SETTINGS_GENERAL_CHOOSE_LANGUAGE_HEADER => "SETTINGS_GENERAL_CHOOSE_LANGUAGE_HEADER".Translate();
+        public static string SETTINGS_GENERAL_RESTART_REQUIRED_TEXT => "SETTINGS_GENERAL_RESTART_REQUIRED_TEXT".Translate();
+        public static string SETTINGS_GENERAL_MORE_INFO_LINK => "SETTINGS_GENERAL_MORE_INFO_LINK".Translate();
+        public static string SETTINGS_GENERAL_MORE_INFO_BUTTON_TEXT => "SETTINGS_GENERAL_MORE_INFO_BUTTON_TEXT".Translate();
+        public static string SETTINGS_GENERAL_ACCESSIBILITY_MORE_INFO_BUTTON_TEXT => "SETTINGS_GENERAL_ACCESSIBILITY_MORE_INFO_BUTTON_TEXT".Translate();
+        public static string SETTINGS_GENERAL_DA => "SETTINGS_GENERAL_DA".Translate();
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/ViewModels/SettingsGeneralViewModel.cs:24]: Just delete this Danish string

**Review by Frederik-S-N-dk — CHANGES_REQUESTED** [review]

**Review by jonasofrich — CHANGES_REQUESTED** [review]
Should also remove the dialog on iOS telling the user to restart the application

**Comment by nc-kano** [comment]
@jonasofrich no, the dialog is still needed but we need to change text

File: NDB.Covid19/NDB.Covid19/ViewModels/SettingsGeneralViewModel.cs (hunk 1 above)
Comment (nc-kano) [reply NDB.Covid19/NDB.Covid19/ViewModels/SettingsGeneralViewModel.cs]: done

File: NDB.Covid19/NDB.Covid19.iOS/Views/Settings/SettingsPageGeneral/SettingsPageGeneralSettingsViewController.cs [hunk 2]
```diff
@@ -111,17 +121,14 @@ void SetupRadioButtons()
 
         public void SwitchValueChanged(object sender, EventArgs e)
         {
-            System.Diagnostics.Debug.Print("Switch clicked. Is on " + switchButton.On);
+            Debug.Print("Switch clicked. Is on " + switchButton.On);
```
[the remark this answers is not in the document]
Comment (nc-kano) [reply NDB.Covid19/NDB.Covid19.iOS/Views/Settings/SettingsPageGeneral/SettingsPageGeneralSettingsViewController.cs]: done

**Review by jonasofrich — APPROVED** [review]

**Review by Frederik-S-N-dk — APPROVED** [review]

**Pull request merged**